plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -28,20 +28,27 @@
 from mongoengine import NotUniqueError
 
 from pulp.plugins.util.publish_step import PluginStep, GetLocalUnitsStep
+from pulp.common import error_codes as pulpcore_error_codes
```

Comment:
can we stick for now with pulp instead of pulpcore? pulpcore is a pulp3 new terminology and concept, i would not like to introduce confusion.